package.json

```diff
@@ -57,14 +57,13 @@
     "@sentry-internal/rrweb-player": "2.12.0",
     "@sentry-internal/rrweb-snapshot": "2.12.0",
     "@sentry/babel-plugin-component-annotate": "^2.16.0",
-    "@sentry/core": "^7.111.0",
-    "@sentry/integrations": "^7.111.0",
-    "@sentry/node": "^7.111.0",
-    "@sentry/react": "^7.111.0",
+    "@sentry/core": "^8.0.0-beta.4",
+    "@sentry/node": "^8.0.0-beta.4",
+    "@sentry/react": "^8.0.0-beta.4",
     "@sentry/release-parser": "^1.3.1",
     "@sentry/status-page-list": "^0.1.0",
-    "@sentry/types": "^7.111.0",
-    "@sentry/utils": "^7.111.0",
+    "@sentry/types": "^8.0.0-beta.4",
+    "@sentry/utils": "^8.0.0-beta.4",
     "@spotlightjs/spotlight": "^1.2.13",
```

Comment:
@Lms24 do we need to bump spotlight as well? 🤔

Reply:
yup to `2.0.0-alpha.1` for now